scores/2023-02-22-alan-wake-ii.md

```diff
@@ -0,0 +1,195 @@
+---
+title: HippoPenny Score
+layout: default
+permalink: /meta-score/alan-wake-ii
+show_excerpts: true
+sort_order: reverse
+sort_by: date
+author_profile: false
+---
+
+<style>
```

Comment:
why don't you create a layout file and put all these configs in that file. These .md files should simply set `layout` (line 3).